guides/common/modules/con_installing-and-configuring-insights-iop.adoc

```diff
@@ -0,0 +1,14 @@
+[id="installing-and-configuring-insights-iop"]
+= Installing and configuring {insights-iop}
+
+:FeatureName: {insights-iop}
+include::snip_technology-preview.adoc[]
+
+{insights-iop} analyzes system health and configuration by applying predefined rules to a small set of local data, such as installed packages, running services, and configuration settings.
+When you install {insights-iop} locally, you can generate {Insights} recommendations without sending system data to Red{nbsp}Hat services.
```

Comment:
Should we mention the vulnerability stuff as well?

Reply:
@Lennonka I don't know if the vulnerability needs to be mentioned here since you are addressing it in #3938 . @chris1984 said it can be addressed in Managing hosts in that PR by you.